src/com/google/javascript/jscomp/DefaultPassConfig.java

```diff
@@ -570,8 +570,13 @@ private void addModuleRewritingPasses(List<PassFactory> checks, CompilerOptions
       passes.add(j2clUtilGetDefineRewriterPass);
     }
 
+    /*
     if (options.instrumentForCoverage) {
       passes.add(instrumentForCodeCoverage);
+    }*/
+
+    if(options.instrumentForCoverageOption != null && options.instrumentForCoverageOption != InstrumentOption.NONE){
```

Comment:
Can instrumentForCoverageOption be null? It would be good if we could guarantee it's always set to some value.